pkg/operator/operator.go

```diff
@@ -100,6 +102,9 @@ func NewOperator() (context.Context, *Operator) {
 
 	// Inject settings from the ConfigMap(s) into the context
 	ctx = injection.WithSettingsOrDie(ctx, kubernetesInterface, apis.Settings...)
+	if s := settings.FromContext(ctx); s != nil {
+		opts.MergeSettings(s)
```

Comment:
nit: should we just always merge the settings and let the `opts.MergeSettings` function just handle the nil settings?

Reply:
I don't have a strong opinion either way, that's fine with me.